src/cephadm/cephadm

```diff
@@ -5155,12 +5155,7 @@ def command_bootstrap(ctx):
     # Notify the Dashboard to show the 'Expand cluster' page on first log in.
     cli(['config-key', 'set', 'mgr/dashboard/cluster/status', 'INSTALLED'])
 
-    logger.info('You can access the Ceph CLI with:\n\n'
-                '\tsudo %s shell --fsid %s -c %s -k %s\n' % (
-                    sys.argv[0],
-                    fsid,
-                    ctx.output_config,
```

Comment:
I think the general idea behind this change is good, but I do worry slightly that if the output_config and output_keyring are not emitted into this log somewhere it might be hard for our hypothetical multi-cluster user to find them again.
Perhaps we should keep the changes regarding the simplified command but output the paths for the config and keyring on their own lines too?

Reply:
I'm somewhat mixed here. In the case of a multi-cluster setup it is somewhat nice to have the fail-safe way to access to shell listed, where I suspect a user might discover these options are not always needed on their own

Reply:
@rkachach since people are making a valid point that is could still be useful to print out the safe, longer command that includes the location of the config and keyring, do we perhaps instead want to print out the long form command always but if we detect there's only a single cluster include an additional message with the shorter one? Something like
`Or, if only one ceph cluster inhabits this host and you are using the default fsid and config location: "sudo {sys.argv[0]} shell"` Maybe you could word it better than I did there but you get the idea.

Reply:
I've updated the code so now we detect if we only have one cluster we show the "short command" and in case of multiple clusters we show the large/complete cephadm shell cmd. I think this way we easily adapt the command depending on the current configuration covering all the case. So in general (1 cluster setup) the user will get the "short command" but in case user has more clusters then he will get the large command.

Reply:
I've updated the code to show both the short and large message.